### PR TITLE
Adds updated Social Cards

### PR DESCRIPTION
**Pull request by ricardobaeta**
Signed-off-by: Ricardo Baeta <ricardo@ricardobaeta.com>

### CODE REVIEW

File: hugo/content/_index.md [hunk 1]
```diff
@@ -3,4 +3,6 @@ type: index
 section: index
 url: /
 title: Machine Learning for Large Scale Code Analysis
+socialImage: img/logo-bg-blue.png
```
Comment (dpordomingo) [inline hugo/content/_index.md]: `logo-bg-blue.png` logo blue is already the default one for the whole landing (according to [`hugo.config.yaml::socialImage`](https://github.com/src-d/landing/blob/master/hugo.config.yaml#L43)), so it's not needed to make it explicit.
```suggestion
```

File: hugo/content/community.md [hunk 2]
```diff
@@ -4,5 +4,6 @@ section: community
 url: community
 title: Community
 socialDescription: Join our Community and contribute and collaborate with other developers
-socialImage: img/logos/logo-bg-white.png
+socialImage: img/logo-bg-blue.png
```
Comment (dpordomingo) [inline hugo/content/community.md]: in order to use our default image, we can just delete this line asking for a different one :dagger: 
```suggestion
```

File: hugo/content/engine.md [hunk 3]
```diff
@@ -4,4 +4,6 @@ section: engine
 url: engine
 title: source{d} Engine
 socialDescription: Powerful language-agnostic analysis of your source code and git history.
+socialImage: img/engine-social-card.png
```
Comment (dpordomingo) [inline hugo/content/engine.md]: I'd create new logos image into [`static/img/logos`](https://github.com/src-d/landing/tree/master/static/img/logos), so they'll be all in the same dir.
```suggestion
socialImage: img/logos/engine-social-card.png
```

File: hugo/content/lookout.md [hunk 4]
```diff
@@ -4,4 +4,6 @@ section: lookout
 url: lookout
 title: source{d} Lookout
 socialDescription: A service for assisted code review, that allows running custom code Analyzers on pull requests.
+socialImage: img/lookout-social-card.png
```
Comment (dpordomingo) [inline hugo/content/lookout.md]: I'd create new logos image into [`static/img/logos`](https://github.com/src-d/landing/tree/master/static/img/logos), so they'll be all in the same dir.
```suggestion
socialImage: img/logos/lookout-social-card.png
```

**Comment by ricardobaeta** [comment]
Thanks @dpordomingo ! All requests were addressed. We can ship this, whenever you have the availability to do so :)

**Pull request merged**